deploy/helm/edot-collector/kube-stack/managed_otlp/values.yaml

```diff
@@ -16,7 +16,7 @@ crds:
   create: true # Install the OpenTelemetry Operator CRDs.
 defaultCRConfig:
   image:
-    repository: "docker.elastic.co/beats/elastic-agent"
+    repository: "docker.elastic.co/beats/elastic-otel-collector"
```

Comment:
I think these might be used for demos and are the reference for how to use EDOT, so once this merges but before the first snapshot or release with these images are available this might be broken.

Reply:
Updating them like this is definitely what we want to do, but this has some chance of causing a bit of chaos until the unified release changes we need actually happen everywhere

Reply:
ok so if I decode correctly the above, we leave the manifests intact but we extend only the tests to use the new images?

Reply:
In this PR, yes. We should update these manifests, but only after there is a publicly available docker image to point them at. Otherwise, these charts will not work by default from main for however long it takes for the unified release job to publish the new container images.

Reply:
Undo this change here and we'll do it as a follow up.

Reply:
ok no image changes in deploy. For now we do the necessary (kinda hacky for the env vars) changes only in our tests and we test both `edot-collector` and `edot-collector-wolfi`. Feel free to unresolve if you spot anything else 🙂